app/controllers/EditionsController.scala

```diff
@@ -0,0 +1,32 @@
+package controllers
+
+import logging.Logging
+import model.forms.CreateIssue
+import play.api.libs.json.Json
+import services.editions.{EditionTemplates, EditionsDB}
+
+import scala.concurrent.ExecutionContext
+
+class EditionsController(db: EditionsDB, val deps: BaseFaciaControllerComponents)(implicit ec: ExecutionContext) extends BaseFaciaController(deps)  with Logging {
+
+  def postIssue = AccessAPIAuthAction(parse.json[CreateIssue]) { req =>
+    val form = req.body
+
+    EditionTemplates.generateEditionTemplate(form.name, form.publishDate).map { template =>
+      db.insertIssue(form.name, form.publishDate, template, req.user)
+      Created
```

Comment:
Can we send back what we've just written here in the response? It'll save us another trip to getIssue :)